static/app/constants/index.tsx

```diff
@@ -329,7 +329,7 @@ export const DATA_CATEGORY_INFO = {
   [DataCategoryExact.PROFILE_DURATION]: {
     name: DataCategoryExact.PROFILE_DURATION,
     apiName: 'profileSeconds',
-    plural: 'profileSeconds',
+    plural: 'profileDuration',
     displayName: 'profile seconds',
     titleName: t('Profile Seconds'),
```

Comment:
The apiName is `profileDuration` and we display `profile hours`.
See https://github.com/getsentry/getsentry/blob/master/getsentry/billing/utils/data_category.py#L209-L218